### ISSUE TITLE
The input about the evaluation_PersonaChat.py

### ISSUE THREAD

**Issue by znxnx** [Chenrj233/LMEDR#6]
Hello, I still have a question, that is, I did not find the input txt file in the evaluation_PersonaChat.py file. If I want to change the input, how can I change it, thank you!

**Comment by Chenrj233** [comment]
The path of the input file will be output like `datapath: file_path` after running the eval script. If you want to test on your own dataset, you should refer to the ParlAI documentation or `eval_PersonaChat_build.py` and construct the input data, then calls `model.generate()` to generate responses

**Comment by Zhouziyi828** [comment]
The path of the input file is 'lib/python3.9/site-packages/parlai-0.1.0-py3.9.egg/data' and the datatype is 'valid'. Then what is the exact test file and how to change it? Because there are many files in the data path.

**Comment by Zhouziyi828** [comment]
Do you mean that if I want to use my own dataset, I should call the model.generate() just like the function in dstc_generate.py?

**Comment by Chenrj233** [comment]
Yes, if you want to test on your own data set, you can refer to the data processing functions in  `eval_PersonaChat_build.py`, `build_dstc.py` and `eval_PersonaChat_build.py`, then use `model.generate()` to generate responses.

**Comment by Zhouziyi828** [comment]
I noticed that the there is a line in the evaluation_PersonaChat.py:
self.history = self.history[(-2*self.args.max_history+1):]

Is the true response included as well if the self.history is cut like this? 
Is "self.history = self.history[(-2*self.args.max_history):-1]" the better way?
